### DISCUSSION TITLE
Remove Podcast episodes

### DISCUSSION THREAD

**Discussion by LilJay86** [advplyr/audiobookshelf#835] [Q&A] [answered]
Hello, i have removed 2 Podcast Episodes and checkt "hard delete file"
The 2 Episodes where removed from the File System and on ABS.
After 1 - 2 hours were these 2 Episodes back onFile System and ABS.

Does anyone know what I'm doing wrong. I suspect that the episodes will be recognized as new and re-downloaded again.
How can I prevent already heard and deleted episodes from being recognized as new? Thanks

**Comment by advplyr** [comment, marked as answer]
That's not supported yet, I hadn't even thought of it. I'll make a feature request out of this

**Reply by advplyr** [reply]
https://github.com/advplyr/audiobookshelf/issues/836
